src/ripple/app/main/GRPCServer.cpp

```diff
@@ -780,9 +780,9 @@ GRPCServer::start()
     if (running_ = impl_.start(); running_)
     {
         thread_ = std::thread([this]() {
-            beast::setCurrentThreadName("rippled : GRPCServer");
+            this_thread::set_name("rippled : GRPCServer");
             // Start the event loop and begin handling requests
-            beast::setCurrentThreadName("rippled: grpc");
+            this_thread::set_name("rippled: grpc");
```

Comment:
Is there a point to calling `this_thread::set_name` twice here?

Reply:
Not to my knowledge.  However I didn't question it just in case there was something subtle I was missing (such as another thread watching).

That being said, a subsequent search showed zero uses of "get thread name", so I don't see how this could be anything but a harmless bug.  I wouldn't object to fixing it as a drive-by.

Reply:
+1 for the drive-by fixing :-)!